src/lib/viewers/media/DashViewer.js

```diff
@@ -30,14 +31,15 @@ class DashViewer extends VideoBaseViewer {
 
         this.api = options.api;
         // Bind context for callbacks
-        this.loadeddataHandler = this.loadeddataHandler.bind(this);
         this.adaptationHandler = this.adaptationHandler.bind(this);
-        this.shakaErrorHandler = this.shakaErrorHandler.bind(this);
-        this.requestFilter = this.requestFilter.bind(this);
+        this.bufferingHandler = this.bufferingHandler.bind(this);
```

Comment:
What do you think about using `handleX` for all new handlers? So `handleBuffering` in this case?